core/primitives/src/shard_layout.rs

```diff
@@ -186,6 +422,52 @@ impl ShardLayout {
         )
     }
 
+    /// Returns the simple nightshade layout, version 4, that will be used in
+    /// production. It adds a new boundary account splitting the "game.hot.tg"
+    /// shard into two smaller shards. This is the first layout used in the
+    /// Instant Resharding and it is the first one where the shard id contiguity
+    /// is broken.
+    ///
+    /// TODO(resharding) Determine the shard layout for v4.
+    /// This layout is provisional, the actual shard layout should be determined
+    /// based on the fresh data before the resharding.
+    pub fn get_simple_nightshade_layout_v4() -> ShardLayout {
```

Comment:
Marketing nit: what about dropping `simple` and saying `get_nightshade_layout_v4` since now?
The ultimate version may be called `get_dynamic_nightshade_layout`

Reply:
Technically speaking this particular layout will not be dynamic so I wouldn't want to call it that quite yet. I called it simple for consistency and greppability. I'm not sure the codebase is the best place for marketing but if you like it more I'm happy to rename it to `get_nightshade_layout_v4`.